addons/website/views/website_templates.xml

```diff
@@ -1249,14 +1249,14 @@
 <template id="option_header_brand_logo" inherit_id="website.placeholder_header_brand" name="Header Brand Logo" active="True">
     <xpath expr="//*[@id='o_fake_navbar_brand']" position="replace">
         <a href="/" t-attf-class="navbar-brand logo #{_link_class}">
-            <span t-field="website.logo" t-options="{'widget': 'image'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            <span t-field="website.logo" t-options="{'widget': 'image', 'class' : 'o_logo'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
```

Comment:
Strange to have `.logo .o_logo`, unify them

Reply:
`.logo` is already used in different places
https://github.com/odoo/odoo/blame/master/addons/website/static/src/img/snippets_options/header_template_centered_logo.svg#L16